ci/docker/runtime_functions.sh

```diff
@@ -854,6 +854,27 @@ nightly_test_javascript() {
     make -C /work/mxnet/amalgamation libmxnet_predict.js MIN=1 EMCC=/work/deps/emscripten/emcc
 }
 
+#Tests Model backwards compatibility on MXNet
+nightly_model_backwards_compat_test() {
+    set -ex
+    export PYTHONPATH=./python/
+    pip install boto3
+    ./tests/nightly/model_backwards_compatibility_check/model_backward_compat_checker.sh
+}
+
+#Backfills S3 bucket with models trained on earlier versions of mxnet
+nightly_model_backwards_compat_train() {
+    set -ex
+    export PYTHONPATH=./python/
+    VENV=mbcc_py2_venv
+    virtualenv -p `which python2` $VENV
+    source $VENV/bin/activate
+    pip install boto3
```

Comment:
Please note (and add a comment or move it somewhere else) that this script can not be called within our regular docker environment because the IAM roles do not get propagated into the container as of now. Also, we have security restrictions here that will have to be addressed before we can bring this into production. I'll cover that later.

Reply:
Done.

Reply:
It's still there, what did you change?

Reply:
I added a comment indicating that it should not be called from the regular docker environment.

Reply:
Sorry, retrospectively, my previous comment was not very clear. Please excuse that.

Everything inside runtime_functions.sh is intended to be run within docker **only**. Please have a look at https://github.com/apache/incubator-mxnet/blob/master/tests/nightly/broken_link_checker_test/JenkinsfileForBLC#L57 to see how to combine the usage of Docker together with the integration of boto to fulfil that constraint. It basically boils down to doing the core logic inside the Docker environment and then running a second script (in this case it's pure shell, but you can also create a second python script dedicated to the publishing process). Feel free to hit me up if you need further advice here.

Reply:
I'm a little confused about it. So this is indeed a couple of lines of shell script which can be easily added in the docker file for MBCC, but how do I install boto on the Jenkins machines since the training script would not be running inside a docker container ?

Reply:
Well, now that I have another look at https://github.com/apache/incubator-mxnet/blob/master/tests/nightly/broken_link_checker_test/JenkinsfileForBLC#L57 I get the interaction with S3.

Reply:
Done in 1f63941e9ade0eaa2f4361ed5877bc97defcf6fb